api/infrastructure/handler/arttoy.handler.go

```diff
@@ -0,0 +1,82 @@
+package handler
+
+import (
+	"net/http"
+
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"shanepee.com/api/apperror"
+	"shanepee.com/api/domain"
+	"shanepee.com/api/service"
+)
+
+type ArtToyHandler struct {
+	artToySvc service.ArtToyService
+}
+
+func NewArtToyHandler(artToySvc service.ArtToyService) ArtToyHandler {
+	return ArtToyHandler{
+		artToySvc,
+	}
+}
+
+// @Summary      Create Art toy
+// @Description  Create a new art toy record
+// @Tags         Art toy
+// @Produce      json
+// @Param        body  body  domain.ArtToyCreateBody  true  "body of Art toy to be created"
+// @Success      200   {object}  domain.ArtToy
+// @Failure      400   {object}  ErrorResponse
+// @Failure      500   {object}  ErrorResponse
+// @Router       /v1/arttoy [post]
+func (h *ArtToyHandler) CreateArtToy(c *gin.Context) {
+	var body domain.ArtToyCreateBody
+	if err := c.ShouldBind(&body); err != nil {
+		handleError(c, apperror.ErrBadRequest("Invalid body"))
+		return
+	}
+
+	artToy := domain.NewArtToy(body.Name, body.Description, body.Price, body.Photo, body.OwnerId)
+	err := h.artToySvc.CreateArtToy(c, artToy)
+	if err != nil {
+		handleError(c, err)
+		return
+	}
+
+	c.JSON(http.StatusOK, artToy)
+}
```

Comment:
```suggestion
// @Summary      Create Art toy
// @Description  Create a new art toy record
// @Tags         Art toy
// @Produce      json
// @Param        body  body  domain.ArtToyCreateBody  true  "Body of Art toy to be created"
// @Success      200   {object}  domain.ArtToy
// @Failure      400   {object}  ErrorResponse
// @Router       /v1/arttoy [post]
func (h *ArtToyHandler) CreateArtToy(c *gin.Context) {
	var body domain.ArtToyCreateBody
	if err := c.ShouldBind(&body); err != nil {
		handleError(c, apperror.ErrBadRequest("Invalid body"))
		return
	}

	artToy := domain.NewArtToy(body.Name, body.Description, body.Price, body.Photo, body.OwnerId)
	err := h.artToySvc.CreateArtToy(c, artToy)
	if err != nil {
		handleError(c, err)
		return
	}

	c.JSON(http.StatusOK, artToy)
}
```
5xx is implicit on every request.